connectors/src/connectors/webcrawler/temporal/activities.ts

```diff
@@ -133,6 +133,18 @@ export async function crawlWebsiteByConnectorId(connectorId: ModelId) {
             type: "pre_navigation",
           });
 
+          const response = await fetch(crawlingContext.request.url, {
+            method: "HEAD",
+            redirect: "manual",
+          });
+
+          if (response.status >= 300 && response.status < 400) {
+            const redirectUrl = response.headers.get("location");
```

Comment:
That would only work for 1 step of redirection right?

Reply:
Yes! Need to loop on that